### DISCUSSION TITLE
Github now disallows opening images in issues in private repos

### DISCUSSION THREAD

**Discussion by drewmiranda-gl** [community/community#67228] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

Github appears to have made a change that has broken the ability to view or download images contained in github issues. I think this may only pertain to private repos but not certain.

While the image is visible inline in the issue text/comment, if the image is large, its impossible to enlarge it, either by opening in new tab or downloading it. It always reports 404 (which i assume is actually access denied and something is wrong with how the URL works with the appended token)

I performed a cursory search on the internet and other than finding many others reporting the same issue, cannot find any workarounds or solutions. Not sure if this has been broken on purpose or if this is a bug.

![image](https://github.com/community/community/assets/107503402/6873f448-a7e7-4063-a62f-6dbc921ca8ff)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by honestmk99** [comment]
Hello
It's my opinion about this issue.

It is difficult for me to provide a definitive answer without access to real-time information or specific knowledge of recent changes made by GitHub. However, based on the details you provided, it seems like there may be an issue with viewing or downloading images in GitHub issues, especially in private repositories.
If multiple users are reporting the same problem and there are no clear workarounds or solutions available, it is likely that this issue is a bug rather than an intentional change. Bugs can occur in software systems, including popular platforms like GitHub, and can impact certain functionalities.
To address this issue, I recommend reaching out to GitHub support or posting about the problem in the GitHub Community Forum. They will have the most accurate information and can provide you with guidance on how to address the problem or inform you if it is a known issue that is being actively worked on.
Remember that GitHub's support channels are the best source of assistance for specific issues you encounter on their platform.

Thank you for your time.

Best

**Comment by 1atAlcone** [comment]
I could understand and say through Legal terms but not in technology terms. Gets my head a Lil bit more into knowing it.

**Comment by dacook** [comment]
This is still an issue for me too. It's frustrating because you can't use browser zoom to increase the image size (the sidebar gets in the way).

Did anyone contact support and get any response?

**Comment by dacook** [comment]
I assume it's a server change that was made to prevent misuse (embedding images being embedded on other sites). But unfortunately it prevents proper use too :(

Perhaps an alternative would be to implement an image zoom with Javascript (although having the freedom to download or open images in another tab is ideal)

**Comment by Fuzzycc** [comment]
Bumping this.

I've uploaded images in a table format in a comment under a repo's issues, the end result was small images in a table. I click to open the image in a new tab... don't work... copy pasting the image to paint but this is an ugly solution lol.
